CONTRIBUTING.md

```diff
@@ -209,6 +209,10 @@ This means that references to implementation details are not appropriate, leave
 It is the behavior change which is important.
 Changes which don't affect users, like changes in tests or in tools which are not packaged, should *not* have a changelog entry.
 
+All changelog entries should also include a reference to a ticket:
```

Comment:
Note that it's not obligatory to type letters `Tickets`. You can just include `CFE-1234` somewhere in commit message text, like this:
```
Changelog: Fix issue mentioned in CFE-1234 and other places.
```

Reply:
I didn't know that! Hmm.

Reply:
Me neither!